index.js

```diff
@@ -109,7 +120,12 @@ TChannel.prototype.listen = function () {
 		throw new Error('Missing server port.');
 	}
 
-	this.serverSocket.listen(this.port, this.host);
+	if (options.fd >= 0) {
+		this.listenFd = options.fd;
```

Comment:
I followed the model of other debuggability stuff I found in the v2 code; it has no immediate use, and I'm happy to axe if it needed. I thought that it may well be useful in live debugging, since it's the only surviving indication that a channel was spawned via FD instead of normal listen.